src/main.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+
+
+import '../style/main.scss';
+
+;let searchFormBoard = 'cats';
+let searchFormLimit = 5;
+
+const apiUrl = `http://reddit.com/r/${searchFormBoard}.json?limit=${searchFormLimit}`;
+
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  render() {
+    return (
+      <div>
+        <h3>Search Form</h3>
+        <form onSubmit={this.handleSubmit()}>
```

Comment:
In the future, no need to invoke `handleSubmit` methods when they are being used as event handlers. The invocation is implied. See me if you're confused.